### ISSUE TITLE
Shortcuts for Find Next and Find Previous do not work

### ISSUE THREAD

**Issue by Okeanos** [uranusjr/macdown#343]
Each time I want to use Find Next (⌘G, cmd+g) or Find Previous (⇧⌘G, shift+cmd+g) it doesn't work and I have to manually click "next" or "previous" in the search bar.

Jump to Selection, however, works fine as a keyboard shortcut.

**Comment by ohthetrees** [comment]
I noticed this too. Oddly, if you close the search dialogue, the keystrokes work again.

**Comment by Jmuccigr** [comment]
Yeah, it looks like the find field grabs the focus, eating up the keyboard commands. So you have to shift focus to the text area. If you click in the text area or close the find field with "Done", then the commands work.

**Comment by ZevEisenberg** [comment]
+1 would love to see this fixed. Command-G works in Safari, Xcode, and other native apps while the search field has focus. Is MacDown doing something non-standard?

**Comment by franklinyu** [comment]
> I have to manually click "next" or "previous" in the search bar.

While I agree that <kbd>⌘</kbd> <kbd>G</kbd> should have worked, I don't think you *have to* use mouse. You can also hit <kbd>Return ↩</kbd> to find next, or <kbd>Shift ⇧</kbd> <kbd>Return ↩</kbd> to find previous.

**Comment by eclab** [comment]
It's worth mentioning that Find-Next (and I presume Find-Previous) still do not work.  Surely this is a simple bug to fix, and it's going on ten years now.

**Comment by ZevEisenberg** [comment]
The last commit merged to ~main~ master (tell me you have an old repo without telling me you have an old repo) was 4 years ago, as of this writing. I miss MacDown every time I open VSCode to edit Markdown 😭
